rest-api-spec/src/main/resources/rest-api-spec/test/snapshot.status/10_basic.yml

```diff
@@ -11,7 +11,39 @@ setup:
 
 ---
 "Get snapshot status":
+  - skip:
+      version:     " - 6.99.99"
+      reason:      "backporting in progress: https://github.com/elastic/elasticsearch/pull/29602"
+  - do:
+      indices.create:
+        index: test_index
+        body:
+          settings:
+            number_of_shards:   1
+            number_of_replicas: 0
 
+  - do:
+      snapshot.create:
+        repository: test_repo_status_1
+        snapshot: test_snapshot
```

Comment:
I'm not sure if the tests will pass when you create a snapshot with name `test_snapshot` in this test and then another snapshot with same name in the other test.

Reply:
reasonable point - but for some reason it passes locally - I think it would be better to use another name explicitly

Reply:
I know why - `ESClientYamlSuiteTestCase` uses each test scenario at the same isolation way as   scenarios in a different yml files - https://github.com/elastic/elasticsearch/blob/a17d6cab9853ae8a87a52210cdd84cfaec8b0ad2/test/framework/src/main/java/org/elasticsearch/test/rest/yaml/ESClientYamlSuiteTestCase.java#L167

Reply:
TIL